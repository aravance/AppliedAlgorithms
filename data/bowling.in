9/1/62275/13XX0/5/1
549/2/X
8/XXXYX8/X3545
50322/05727104104413
XXX/XXX6/XXXX
XXX51X7/X813X5/
XX9/XX62X6/XXXX
9
4/XX9/X7/X46X08
619/33902313146/8113
XX47XXXXXXX9/
XXXXXXX6/X
23XXXX62X4/120/X
XXXX35X1/5/XX
XX3/XXX412733XX
X2/81X907/3445X2/1
XXX8/XXXXXXXXX
XXX3/X36X6/XX2
X6/XXXXXXXX6/
4/X9/X80XX31XX1X
9/6/XXXXX3/XX55
0/XXXXX1/XXXX6
XXX5/XXX3/7/X1/-
729/435231815342903
X16XXX6/X1/XX27
234242188031635/43728
62X81X8/31X8/7/6/
XXXXXXXXXXXX
X1/51722012XXX3/1/
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
00000000000000000000
*
