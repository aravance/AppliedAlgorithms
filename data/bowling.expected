125
INVALID
INVALID
59
INVALID
INVALID
221
INVALID
INVALID
80
INVALID
INVALID
171
INVALID
INVALID
135
INVALID
INVALID
266
INVALID
INVALID
257
INVALID
INVALID
201
INVALID
INVALID
300
INVALID
INVALID
0
